frontend/src/component/SubmitForm.css

```diff
@@ -1,30 +1,26 @@
-.form-add {
+.form-submit {
     display: flex;
     flex-flow: column;
     align-items: center;
     justify-content: space-between;
     gap: 32px;
-    height: 50vh;
-    width: 50vh;
+    height: 40vh;
+    width: 40vh;
```

Comment:
I suggest using `vw` as unit for width. `vh` is percentage relative to the viewport height, where `vw` is percentage of the viewport width, which is normally more relevant for a width attribute :)